ddd/set.rb

```diff
@@ -0,0 +1,15 @@
+class Set
```

Comment:
This class isn't used yet?

Reply:
no, not yet, it's just 'put on the shelf' here for later...

Reply:
it's used not ;oP